commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/tar/TarFileSystem.java

```diff
@@ -16,16 +16,6 @@
  */
 package org.apache.commons.vfs2.provider.tar;
 
-import java.io.File;
```

Comment:
Don't mess with imports, please.

Reply:
sorry about that.  I understand completely.  IDE did some of this automatically.  I will undo it.